### PR TITLE
update to include parsing for interface in alert

### PR DESCRIPTION
**Pull request by weslambert**

### CODE REVIEW

File: configfiles/1033_preprocess_snort.conf [hunk 1]
```diff
@@ -7,7 +7,10 @@ filter {
   if [type] == "snort" {
     # This is the initial parsing of the log
     grok {
-      match => ["message", "\[%{INT:gid}:%{INT:sid}:%{INT:rev}\]\s%{DATA:alert}\[Classification:\s+%{DATA:classification}\]\s+\[Priority:\s+%{INT:priority}\]:\s+{%{DATA:protocol}}\s+%{IPV4:source_ip}:%{INT:source_port}\s+->\s+%{IPV4:destination_ip}:%{INT:destination_port}%{GREEDYDATA:message}", "message", "\[%{INT:gid}:%{INT:sid}:%{INT:rev}\]\s%{DATA:alert}\[Classification:\s+%{DATA:classification}\]\s+\[Priority:\s+%{INT:priority}\]:\s+{%{DATA:protocol}}\s%{IPV4:source_ip}\s+->\s+%{IPV4:destination_ip}"]
+      match => ["message", "\[%{INT:gid}:%{INT:sid}:%{INT:rev}\]\s%{DATA:alert}\[Classification:\s+%{DATA:classification}\]\s+\[Priority:\s+%{INT:priority}\]:\s+<%{DATA:interface}>\s+{%{DATA:protocol}}\s+%{IPV4:source_ip}:%{INT:source_port}\s+->\s+%{IPV4:destination_ip}:%{INT:destination_port}",
```
Comment (dougburks) [inline configfiles/1033_preprocess_snort.conf:10]: Thanks for the PR!  Help me understand what's going on here.  If I'm reading it correctly, originally there were 2 matches for 2 different formats and this PR is doubling that to 4 to account for interface or no interface.  Good so far?  If that's the case, should the 1st match have `%{GREEDYDATA:message}"` at the end (just like the 3rd match does)?

**Review by dougburks — CHANGES_REQUESTED** [review]
Please see question on https://github.com/dougburks/elastic-test/pull/83/files/7a2bcd0ce4d79429dfb6ce3f9ad2d9ca3d794c1e#diff-9feaeb86257db23eae37db31a03dd523

Thanks!

**Comment by weslambert** [comment]
I didn't think you had to have it (I think it's just picking up if anything
else is left at the end of the message --seems like it works without it,
unless I'm missing something), but we can add it as well if you think that
would work better.

Thanks,
Wes

**Comment by dougburks** [comment]
I'm totally not a grok expert and I haven't done any testing, so I don't know if either way works better, but I do think that we should be consistent.  So maybe all 4 matches should have `%{GREEDYDATA:message}` OR none of them should have it.

**Comment by weslambert** [comment]
I just removed it and tested -- not receiving any _grokparsefailure(s) at
this time.  Submitted additional commit.

**Comment by dougburks** [comment]
Awesome, thanks Wes!

**Pull request merged**